Analysis/Utkscan/core/source/DetectorDriverXmlParser.cpp

```diff
@@ -44,16 +44,12 @@
 #include "VandleProcessor.hpp"
 
 //These headers are for handling experiment specific processing.
-#include "E11027Processor.hpp"
-#include "TemplateExpProcessor.hpp"
-#include "VandleOrnl2012Processor.hpp"
-
-#ifdef useroot //Some processors REQUIRE ROOT to function
 #include "Anl1471Processor.hpp"
+#include "E11027Processor.hpp"
```

Comment:
Do we still want to carry around experiment specific headers?

Reply:
Well, they have to be there so that the if tree works properly when parsing the XML. If you have another suggestion on how to add a custom processor then I'm all ears.